Learn.md

```diff
@@ -0,0 +1,178 @@
+# Learn.md ✍
+# ConvoForm 
+[ConvoForm.com](https://www.convoform.com/?utm_source=genai_works&utm_medium=social&utm_campaign=github_launch) transforms traditional forms into interactive conversational experiences, powered by AI for an enhanced user journey. Think Google Forms, but more engaging and intuitive.
+
+#### Features:
+
+- **AI-Powered Form Generation:** Automatically generate comprehensive forms by describing your needs, streamlining the creation process.
+- **Real-time Form Editing and Preview:** Edit forms with live changes previewed on the same page, providing immediate feedback.
+- **Customizable Submission Pages:** Tailor the submission page with your organization's branding and personalized messages.
+
+# Table of Contents 
+1. [Introduction](#introduction-)
+2. [Tech Stack](#tech-stack-)
+3. [Contributing](#contributing-)
+   - [Development Workflow](#development-workflow)
+   - [Issue Report Process](#issue-report-process)
+   - [Pull Request Process](#pull-request-process-)
+4. [Setting Up on your machine](#setting-up-on-your-machine-)
+5. . [Resources for Beginners](#resources-for-beginners-)
+   - [Basics of Git and GitHub](#basics-of-git-and-github-)
+6. [Documentation](#documentation-)
+7. . [Code Reviews](#code-reviews-)
+8. . [Feature Requests](#feature-requests-)
+9. [Spreading the Word](#spreading-the-word-)
+
+## Introduction 🖥️
+ConvoForm.com transforms traditional forms into interactive conversational experiences, powered by AI for an enhanced user journey.
+
+## Tech Stack 🗃️
+- **Frontend**: [Next.js](https://nextjs.org) for optimized server and client rendering.
+- **Backend**: [tRPC](https://trpc.io) for type-safe API development.
+- **AI Integration**: [GPT-3.5-Turbo](https://platform.openai.com/docs/models/gpt-3-5-turbo) for dynamic form generation and response analysis.
+- **Real-time updates**: [Socket.io](https://socket.io/) for live form progress tracking.
+
+## Contributing 📝
```

Comment:
We already have `CONTRIBUTING.md` file in the repo. Don't need to add this section here.